Cargo.toml

```diff
@@ -45,9 +46,7 @@ web-sys = { version = "0.3.49", optional = true, features = ["console", "Headers
 console_error_panic_hook = { version = "0.1.7", optional = true }
 
 [dev-dependencies]
-ark-std = { version = "0.4.0", default-features = false }
 bincode = "1.3"
-getrandom = { version = "0.2", features = ["js"] }
```

Comment:
the CI err is probably caused by the removal of this line.

Reply:
Right, I deleted it during experiments and forgot to restore it.